crud/common/const.lua

```diff
@@ -5,6 +5,8 @@ const.RELOAD_SCHEMA_TIMEOUT = 3 -- 3 seconds
 const.FETCH_SHARDING_METADATA_TIMEOUT = 3 -- 3 seconds
 const.SHARDING_RELOAD_RETRIES_NUM = 1
 
+const.GET_SPACE_TIMEOUT = 3 -- 3 seconds
```

Comment:
@Totktonada suggested to sync this one with vshard call timeout:
https://github.com/tarantool/crud/issues/95#issuecomment-968180577 

I think it worth considering. For example, we really may subtract the space wait timeout from the call timeout or we may set get space timeout to be less or equal than this timeout. (We need to think about this before implementing, maybe discuss a bit.)

Reply:
Thanks, I'm thinking about it now. If the proposed version of the loop is suitable, a more complex timeout condition can be implemented there